Makefile

```diff
@@ -106,10 +106,14 @@ LIBRARY_TARGETS += \
 
 TEST_TARGETS += \
 	example_cpp_smart_card_client_app/build/integration_tests \
+	smart_card_connector_app/build/js_to_cxx_tests \
```

Comment:
WDYT about this naming? I'm trying to come up with something more precise than `integration_tests`.